server.js

```diff
@@ -5,6 +5,13 @@ const path = require("path");
 const fs = require("fs");
 const urljoin = require("url-join");
 
+const fetchTwitterImage = require("./src/api/fetchTwitterImage");
```

Comment:
if this is not used by the client it needs to stay in a ./src/server/ folder otherwise it gets included in the bundle